### ISSUE TITLE
Investigate approach for displaying records without a match to your records

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/ServiceNet#570]
We need to start giving users the ability to see records 'owned' by another system account even if they don't have a match to one of their records.

I'm looking to get an understanding of the difficulty of various approaches and how we might handle it most efficiently.

I think the most direct approach might be to add a multi-select filter that allows a user to 'see records for the selected accounts'. This would default to the user's own records only.

So ideally it would work as follows (but im open to feedback if certain things make it much more difficult):

1. My own records with matches still show as they did previously

2. My own records without matches still show as they did previously

3. Records from a partner system that have no other matches should appear similar to how 2 above shows (single record view only, no side-by-side)

4. Records from partner systems with matches in other partner systems (but not my own). Ideally they would show similarly to how 1 above shows. The question is which of the multiple records do we pick as the 'main' record whose name shows on the main screen. Let me know if you have suggestions or if it's easy to just randomly select one.

I think it might also make sense to have the view for 4 jump right to side-by-side view

Some other gotchas:

* Searching and filtering should ideally work as before
* I think geo filtering should be roughly fine, we would just need to pull all options from all locations - the geocoding changes should help here
* The partner source filter may be able to still work - it may just need to be changed to the selected partner is EITHER the main record's owner or one of the matched record's owner

* the service type/taxonomy search will need to be adjusted with the rosetta stone approach we talked about

**Comment by johnhbenetech** [comment]
this is just more thought for #440

**Comment by awalkowiak** [comment]
@johnhbenetech regarding 3. and 4. and showing it on home page. If users don't want to have it in separate page, maybe we could separate it to two tabs. I think this should be easier to implement (looking for overall work required), beacause we can focus on OrganizationMatch on one tab and on Organization on second tab. Something like this:

![Selection_067](https://user-images.githubusercontent.com/12113312/62203878-538c2e80-b38c-11e9-8dfd-d354c93d3940.png)

@pmuchowski @jkondrat @kkaczmarczyk guys what are your thoughts regarding this feature/change? How do you see its complexity and what would be the best approach in your opinion?

**Comment by pmuchowski** [comment]
@awalkowiak it looks fine to me

**Comment by jkondrat** [comment]
@awalkowiak : It looks good, but I don't think that displaying records in a separate tab will make this much easier

**Comment by awalkowiak** [comment]
@jkondrat I guess this gives us option to work only on one endpoint at the time. Either OrganizationMatch or Organization. Do you see any other solution that could be helpful here?

**Comment by johnhbenetech** [comment]
For 3, how would that case be possible by just looking at organizationmatch? Does organizationmatch also have rows for organizations with no matches?

**Comment by awalkowiak** [comment]
No, but I guess that we could add those orgs during activity info building. However then for hiding there would need to add relation to organization.

**Comment by kkaczmarczyk** [comment]
@johnhbenetech Actually, does it make sense to display all of the 'filter-matched' Organization Matches that do not belong to your account? This way there won't be a need to randomly (or by some other way) choose one of the matches as the 'main' one. The clicked one would be the 'main' record.

**Comment by johnhbenetech** [comment]
So for 4, if we had a cluster of 5 records from other systems - each of the 5 records would appear as a row on the main view. Clicking any of those records would allow you to view the other 4 records in the cluster with the clicked record as 'main'?

**Comment by kkaczmarczyk** [comment]
Correct. What are the downsides?

**Comment by johnhbenetech** [comment]
I think that sounds like a good approach

**Comment by kkaczmarczyk** [comment]
Update per Slack discussion: 

- add the checkbox filter option "only show matching" somewhere near Partner selection. 
- It should be checked by default
- checked: would work as it is now - displaying Org with Org Matches within user's account
- by unchecking it and clicking 'Filter' button, user would trigger displaying Org with Org Matches unrelated with user's account (without a match to user's records)
